ospf6d/ospf6_top.c

```diff
@@ -199,6 +204,8 @@ void ospf6_delete(struct ospf6 *o)
 	ospf6_flush_self_originated_lsas_now();
 	ospf6_disable(ospf6);
 
+	ospf6_serv_close();
```

Comment:
What's the reasoning behind moving this? I'm not familiar enough with this code to know.

Reply:
Currently, the socket is opened at the startup and closed at the shutdown.
I moved open/close to opsf6_create/delete respectively, so the socket lives only together with the ospf6 instance which is controlled by the user ([no] router ospf6). If we start listening on a socket before the instance is created but don't read from it - it'll be just a waste of memory.

Reply:
This changes are already merged in master by PR #7058 
Only a default ospf6 instance is created from ospf6_init().

Reply:
Yes, you're right, that's why I created this PR only for 7.5.

Reply:
Okay, 
Got it